src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/PromotionTranslation.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources
+    xmlns="https://api-platform.com/schema/metadata/resources-3.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="https://api-platform.com/schema/metadata/resources-3.0 https://api-platform.com/schema/metadata/resources-3.0.xsd"
+>
+    <resource class="Sylius\Component\Promotion\Model\PromotionTranslation">
+        <operations>
+            <operation class="ApiPlatform\Metadata\NotExposed" uriTemplate="/admin/promotions/{code}/translations/{localeCode}">
```

Comment:
Why the `NotExposed` action? If I remember correctly, we settled in the last meeting that we would do the translation in the same way as it is done for `ShippingMethodTranslation`.

Reply:
I verified that this operation class hides the endpoint from the Swagger UI.